app/views/kuroko2/job_definitions/_list.html.slim

```diff
@@ -21,7 +21,7 @@
           td= labeled_status(definition.job_instances.take)
           td
             - definition.tags.each do |tag|
-              =link_to tag.name, root_path('tag': [tag.name]), class: 'label label-default spacer-right-3'
+              =link_to tag.name, { 'tag': [tag.name] }, class: 'label label-default spacer-right-3'
```

Comment:
Because this partial view used in dashboards and user details pags (`/users/:id`).